server/features/publish.feature

```diff
@@ -0,0 +1,88 @@
+@wip
+Feature: Publish
+
+    @auth
+    Scenario: Publish Event
+        When we post to "/products" with success
+        """
+        {
+            "name":"prod-1","codes":"abc,xyz", "product_type": "both"
```

Comment:
I'm not sure yet about this whole filtering, if filters will be shared or there will be some switch for content type on it or destination. atm it's more like spike to see if we can reuse the superdesk publishing